NashTeamAlpha/BangazonWeb
# Sub product type view model

## Pull Request Description

Include the applicable information from the list below here 
1. touched AllSubProductsTypeViewModel.cshtml
2. Provided code for future View that will display list of sub types of a given type.
## Steps to Test

Include the applicable information from the list below here
1. Review code to make sure it is consistent with similar view models.
## Link to Feature Ticket

Provide the issue number that this pull request fixes
#125
